internal/remediation/remediation.go

```diff
@@ -35,8 +36,16 @@ func (opts RemediationOptions) MatchVuln(v resolution.ResolutionVuln) bool {
 }
 
 func (opts RemediationOptions) matchSeverity(v resolution.ResolutionVuln) bool {
-	// TODO
-	return true
+	maxScore := -1.0
+	// TODO: also check Vulnerability.Affected[].Severity
+	for _, sev := range v.Vulnerability.Severity {
+		if score, _, _ := severity.CalculateScore(sev); score > maxScore {
+			maxScore = score
+		}
+	}
+
+	return maxScore < 0 || // Always include vulns with unknown severities
+		int(10*maxScore) >= int(10*opts.MinSeverity) // CVSS scores are only to 1 decimal place
```

Comment:
Any reason in particvular for this int(10*X) bit? Add a comment.

Reply:
Clarified comment on why (& used `math.Round` instead)